docs/CONFIG-VARS.md

```diff
@@ -274,6 +274,8 @@ V4_CFG_POSTGRES_SERVERS:
 | V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy openldap in the namespace for authentication | bool | false | false | [Openldap Config](../roles/vdm/templates/generators/openldap-bootstrap-config.yaml) | viya |
 | V4_CFG_CONSUL_ENABLE_LOADBALANCER | Setup LB to access consul ui | bool | false | false | Consul ui port is 8500 | viya |
 | V4_CFG_ELASTICSEARCH_ENABLE | Enable opendistro search | bool | true | false | When deploying LTS less than 2020.1 or Stable less than 2020.1.2 set to false | viya |
+| V4_CFG_VIYA_START_SCHEDULE | Configure your SAS Viya deployment to start on specific schedules | string |  | false | This variable accepts [CronJob schedule expressions](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax) to create your Viya start job schedule. Must be used in conjunction with `V4_CFG_VIYA_STOP_SCHEDULE` | viya |
+| V4_CFG_VIYA_STOP_SCHEDULE | Configure your SAS Viya deployment to stop on specific schedules | string |  | false | This variable accepts [CronJob schedule expressions](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax) to create your Viya stop job schedule. Must be used in conjunction with `V4_CFG_VIYA_START_SCHEDULE` | viya |
```

Comment:
Are there samples of this for the end user? I see the links, but having an example file with these values would be helpful.

Reply:
I can add something in the example files for these two variables. I'll go a with a similar route to "Monitoring and Logging" in the existing example files:
https://github.com/sassoftware/viya4-deployment/blob/main/examples/ansible-vars-iac.yaml#L36

Include example variables but comment them out, since the user would not necessarily want to start and stop Viya on schedule by default.

Reply:
Updated example files, ready for re-review.

Reply:
Will take a look. Thx